features/apimgt/org.wso2.carbon.apimgt.admin.feature/src/main/resources/admin/source/src/app/components/ThrottlingPolicies/Create/CreateCustomRulePolicy.js

```diff
@@ -110,18 +123,21 @@ class CreateCustomRulePolicy extends Component {
 
                         <CustomRuleDetails
                             policy={this.state.policy}
+                            state={this.state}
```

Comment:
Why we need to send state object as is, shall we use spread operator and send the state values as props ?
```javascript
<CustomRuleDetails {...this.state} />
```